pyensembl/gtf.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, gtf_path, cache_directory_path=None):
 
     def __eq__(self, other):
         return (
-            isinstance(other, GTF) and
+            (other.__class__ == GTF) and
```

Comment:
Hard to imagine when we'd want a sub-class of GTF to equal to a GTF instance, seemed safer to restrict equality to exactly the same class. .